src/styles/ThemeStylesProvider.tsx

```diff
@@ -2,10 +2,8 @@
 import React, {useMemo} from 'react';
 import useTheme from './themes/useTheme';
 import ThemeStylesContext from './ThemeStylesContext';
-// TODO: Rename this to "styles" once the app is migrated to theme switching hooks and HOCs
-import {stylesGenerator as stylesUntyped} from './styles';
-
-const styles = stylesUntyped;
+// TODO: Replace this import with "styles" once the static style export from "styles.js" isn't used anymore
```

Comment:
I'll create an issue for this instead and we can remove the TODO comment  

https://github.com/Expensify/App/issues/29910